README.md

```diff
@@ -155,28 +155,43 @@ The ⓒ mark means that [`@uppy/companion`](https://uppy.io/docs/companion), a s
   <img src="https://saucelabs.com/browser-matrix/transloadit-uppy.svg" alt="Sauce Test Status"/>
 </a>
 
-We aim to support IE11 and recent versions of Safari, Edge, Chrome, Firefox and Opera.
+We aim to support recent versions of Safari, Edge, Chrome, Firefox and Opera.
 
-We still run end-to-end tests with IE10, but we are not actively supporting it or fixing visual / minor issues.
+We still run end-to-end tests with IE11, but we are not actively supporting it or fixing visual / minor issues.
 
 ### Polyfills
 
-Uppy heavily uses Promises. If your target environment [does not support Promises](https://caniuse.com/#feat=promises), use a polyfill like `es6-promise` before initializing Uppy.
+Here's a list of polyfills you'll need to include to make Uppy work in older browser (e.g., IE11):
```

Comment:
I usually prefer things as "such as" over "e.g." as it reads more pleasant and is less pretentious.

```suggestion
Here's a list of polyfills you'll need to include to make Uppy work in older browsers, such as IE11:
```